lib/core/keys.dart

```diff
@@ -197,54 +197,54 @@ final Map _codeMap = {
   KeyCode.COMMA: ',', //
   KeyCode.SLASH: '/', //
   KeyCode.BACKSLASH: '\\', //
-  KeyCode.SEMICOLON: ";", //
-  KeyCode.DASH: "-", //
-  KeyCode.EQUALS: "=", //
-  KeyCode.APOSTROPHE: "`", //
+  KeyCode.SEMICOLON: ';', //
+  KeyCode.DASH: '-', //
+  KeyCode.EQUALS: '=', //
+  KeyCode.APOSTROPHE: '`', //
   KeyCode.SINGLE_QUOTE: "'", //
   KeyCode.ENTER: 'enter', //
   KeyCode.SPACE: 'space', //
-  KeyCode.TAB: "tab", //
+  KeyCode.TAB: 'tab', //
 
   KeyCode.OPEN_SQUARE_BRACKET: '[', //
   KeyCode.CLOSE_SQUARE_BRACKET: ']', //
 
   KeyCode.LEFT: 'left', //
   KeyCode.RIGHT: 'right', //
-  KeyCode.UP: "up", //
-  KeyCode.DOWN: "down", //
-
-  KeyCode.BACKSPACE: "backsapce", //
-  KeyCode.CAPS_LOCK: "caps_lock", //
-  KeyCode.DELETE: "delete", //
-  KeyCode.END: "end", //
-  KeyCode.ESC: "esc", //
-  KeyCode.HOME: "home", //
-  KeyCode.INSERT: "insert", //
-  KeyCode.NUMLOCK: "numlock", //
-  KeyCode.PAGE_DOWN: "page_down", //
-  KeyCode.PAGE_UP: "page_up", //
-  KeyCode.PAUSE: "pause", //
-  KeyCode.PRINT_SCREEN: "print_screen", //
+  KeyCode.UP: 'up', //
+  KeyCode.DOWN: 'down', //
+
+  KeyCode.BACKSPACE: 'backsapce', //
```

Comment:
O_o

Is that misspelled for a reason, or can we correct it without weirding up something else?

Reply:
I have no idea where this is used. Maybe @devoncarew can shed some light here?